README.md

```diff
@@ -374,3 +379,75 @@ $ git add git_deploy_key.enc .travis.yml
 $ git commit -m "ci(travis): Add the encrypted private ssh key"
 $ git push
 ```
+
+##### Add the SSH private key to Circle CI
+
+First we encrypt the `git_deploy_key` (private key) using a symmetric encryption (AES-256).  Run the folllowing `openssl` command and *make sure to note the output which we'll need later*:
+
+```bash
+$ openssl aes-256-cbc -e -p -in git_deploy_key -out git_deploy_key.enc -K `openssl rand -hex 32` -iv `openssl rand -hex 16`
+salt=711624E085AB48A4
+key=BB44596FDC00CD2B270BF43B3968BC072AEF026EF4A3C870F874449DA15A5295
+iv =B341F7EBC34D8CF51C4237F39F84FF0D
+```
+
+Open `https://circleci.com/gh/GITHUB_NAME/PROJECT_NAME/edit#env-vars` in your browser, **OR**, login to [CircleCI](https://circleci.com/), choose *PROJECTS* from the left sidebar and click on the *SETTINGS* cog button for your project.  Then on the project sidebar, under *BUILD SETTINGS*, choose *ENVIRONMENT VARIABLES*.  **On that page**, click *ADD VARIABLE* and add `SSH_PASSPHRASE` with the value set during the [SSH keys generation](#generate-the-ssh-keys) step:
```

Comment:
Instead of describing the UI steps maybe is better to just ask to something like
```
Add the following [environment variables](https://circleci.com/docs/2.0/env-vars/#adding-environment-variables-in-the-app) to Circle CI:
- `SSH_PASSPHRASE` with with the value set during the [SSH keys generation](#generate-the-ssh-keys)
- `REPO_ENC_KEY` with the value of....
- `REPO_ENC_IV` with the value of....
```

This way it's more consistent with the rest of the doc in terms of style and it avoid unnecessary details about the Circle CI UI specifics. By linking to the Circle CI doc instead, we won't have to update the doc if Circle CI decide decide to put the PROJECTS, BUILD SETTING menu somewhere else in the UI.

Reply:
Great point.  And clearer indeed.